opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -340,27 +340,27 @@ def add_lazy_link(self, link: "trace_api.Link") -> None:
             return
         self.links.append(link)
 
-    def start(self):
+    def start(self, start_time: int = None):
         with self._lock:
             if not self.is_recording_events():
                 return
             has_started = self.start_time is not None
             if not has_started:
-                self.start_time = util.time_ns()
+                self.start_time = start_time or util.time_ns()
```

Comment:
```suggestion
                self.start_time = start_time if start_time is not None or util.time_ns()
```

I don't know why they would, but if someone wants to zero out the start/end time we should let them.